machine.go

```diff
@@ -374,7 +378,12 @@ func (m *Machine) startup(command string, extracontent [][2]string) int {
 	w.WriteFile("/wrapper",
 		fmt.Sprintf(commandWrapper, command), 0755)
 
-	w.WriteFile("/init", initScript, 0755)
+	shebang := "#!/bin/busybox sh"
+	if mergedUsrSystem() {
+		shebang = "#!/usr/bin/busybox sh"
+	}
+	w.WriteFile("/init",
+		fmt.Sprintf("%s\n%s", shebang, initScript), 0755)
```

Comment:
Sure. Just to keep it here:

- initrd creation fails due non-existing /usr/lib/modules.
- initrd couldn't be started since /usr/bin/busybox absence.

Both problems are caused bu initrd generated without merged /usr